packages/venia-ui/lib/components/LegacyMiniCart/product.js

```diff
@@ -13,6 +13,7 @@ import defaultClasses from './product.module.css';
 import ProductOptions from './productOptions';
 import Section from './section';
 import { gql } from '@apollo/client';
+import { useIntl } from 'react-intl';
```

Comment:
Unsure if we need to support translations in legacy components

Reply:
It's legacy, but this component is still used on the site, so it would benefit from having translations.